src/main/java/com/telesign/phoneid/PhoneId.java

```diff
@@ -409,4 +486,148 @@ public PhoneIdLiveResponse live(String phone_number, String ucid, String origina
 
 		return response;
 	}
+
+	/**
+	 * @param phone_number
+	 * @param ucid				[Required] A string specifying one of the Use Case Codes.
+	 * @param originating_ip	[Optional] The end users IP Address. This value must be in the format of IPv4 and IPv6 Addresses.
+	 * @param session_id		[Optional] A free-form string (up to 64 ASCII characters) that indicates an ID (either raw or hashed) 
+	 * 										that is unique to the users session with the customer. 
+	 * 										The session_id links multiple calls and other TeleSign services that are associated with the current session.
+	 * @return PhoneIdSimSwapCheckResponse
+	 */
+	public PhoneIdSimSwapCheckResponse simSwap(String phone_number, String ucid, String originating_ip, String session_id) {

```

Comment:
I noticed that in several places you are introducing new code that does not follow Java naming convention. Is this because you plan to do it all in #35 ?